api/CanvasRenderingContext2D.json

```diff
@@ -115,7 +115,7 @@
           "status": {
             "experimental": true,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
```

Comment:
I was spot checking these and found that `addHitRegion` has several children. Shouldn't they also be marked as deprecated? Though we'll need to make a note of which child features were marked deprecated, so we can update any pages (if needed).

Reply:
Oh, it should probably be checked by linter. I'll filed an issue.